code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Concepts.hs

```diff
@@ -14,9 +14,10 @@ import Control.Lens ((^.))
 ----- Acronyms -----
 
 acronyms :: [CI]
-acronyms = [assumption, centreMass, dataDefn, gamePhysics, genDefn, goalStmt, inModel,
+acronyms = [assumption, centreMass, dataDefn, genDefn, goalStmt, inModel,
   likelyChg, ode, requirement, refBy, refName, srs, thModel, threeD, twoD, typUnc, unlikelyChg]
 
 centreMass, gamePhysics :: CI
 centreMass  = commonIdeaWithDict "centreMass"  (ctrOfMass ^. term) "CM" [physics]
-gamePhysics = commonIdeaWithDict "gamePhysics" (pn "game physics library") "Game Physics" [physics]
+--This is a hack that is used accross the board, we should consider using something 
```

Comment:
I'm not fully sure what the purpose of this comment is; we should consider using something (like what?)

(Also, it should be "across", not "accross")